packages/atomic/src/components.d.ts

```diff
@@ -40,7 +40,7 @@ export namespace Components {
         /**
           * The base path shared by all values for the facet.  Specify the property as an array using a JSON string representation: ```html  <atomic-category-facet base-path='["first value", "second value"]' ></atomic-category-facet> ```
          */
-        "basePath"?: string[];
+        "basePath": string[] | string;
```

Comment:
Is there some way we could only allow javascript/JSX frameworks to pass an array? I prefer not to expose a type to a framework for which it's useless just because we need it internally or for some unrelated framework.